components/tikv_util/src/security.rs

```diff
@@ -97,6 +98,21 @@ impl SecurityConfig {
         }
         Ok((ca, cert, key))
     }
+
+    /// Determine if the cert file has been modified.
+    /// If modified, update the timestamp of this modification.
+    fn is_modified(&self, last: &mut Option<SystemTime>) -> Result<bool, Box<dyn Error>> {
+        let this = fs::metadata(&self.cert_path)?.modified()?;
+        if let Some(last) = last {
+            if *last == this {
+                return Ok(false);
```

Comment:
Any tests?

Reply:
test for updating certs and No coredump happens?

Reply:
Test for updating modified time.

Reply:
Done